scripts/update_components.sh

```diff
@@ -90,7 +90,7 @@ do_update() {
         line_number=$(grep -n '##' CHANGELOG.md | sed -n '2p' | cut -d ":" -f 1)
         result=$((line_number - 2))
 
-        sed ""$result"a\* Updated $1 to revision $last_commit" $CHANGELOG_FILE > $CHANGELOG_TMP_FILE
+        sed "\"$result\"a\* Updated $1 to revision $last_commit" $CHANGELOG_FILE > $CHANGELOG_TMP_FILE
```

Comment:
Testing results showed an error:
```bash
sed: -e expression #1, char 1: unknown command: `"'
```

```suggestion
        sed "$result a\* Updated $1 to revision $last_commit" $CHANGELOG_FILE > $CHANGELOG_TMP_FILE
```

Reply:
5661ca93e27a4b3d8c9f9e0b06bd197d274fbef1